solidity_contracts/src/UniswapV2Router/libraries/UniswapV2Library.sol

```diff
@@ -33,7 +33,7 @@ library UniswapV2Library {
                         hex"ff",
                         factory,
                         keccak256(abi.encodePacked(token0, token1)),
-                        hex"666a5b78ea0b660c426b08cb5b7427447e909408067de1a5519c772ee9a3c032" // init code hash
+                        hex"0f5b822a8dffa6ce589a2c240d78a6a2b38a51835a97ceab40c1f301e46ba30b" // init code hash
```

Comment:
From where this init code hash was taken ?
Did not find it in commit history of https://github.com/Uniswap/v2-periphery/blob/0335e8f7e1bd1e8d8329fd300aea2ef2f36dd19f/contracts/libraries/UniswapV2Library.sol#L24

Reply:
run `forge build`, open `UniswapV2Library.json` and take the keccak hash of the bytecode hexstring